crypto-common/Cargo.toml

```diff
@@ -17,10 +17,10 @@ hybrid-array = "=0.2.0-pre.5"
 
 # optional dependencies
 rand_core = { version = "0.6.4", optional = true }
+getrandom = { version = "0.2", optional = true }
 
 [features]
 std = []
-getrandom = ["rand_core/getrandom"]
```

Comment:
Perhaps:

```toml
getrandom = ["dep:getrandom", "rand_core?/getrandom"]
```

...which would ensure `crypto_common::rand_core::OsRng` is still available when `getrandom` and `rand_core` are both enabled.